comp/otelcol/otlp/components/exporter/serializerexporter/go.mod

```diff
@@ -189,8 +191,8 @@ require (
 	go.opentelemetry.io/otel/sdk/metric v1.22.0 // indirect
 	go.opentelemetry.io/otel/trace v1.23.1 // indirect
 	go.uber.org/atomic v1.11.0 // indirect
-	go.uber.org/dig v1.17.0 // indirect
-	go.uber.org/fx v1.18.2 // indirect
+	go.uber.org/dig v1.17.1 // indirect
+	go.uber.org/fx v1.21.0 // indirect
```

Comment:
I believe we should keep this dependencies version the same across all Go modules. Can we revert back this change?

Reply:
I did a find replace and a full mod tidy - should be fixed now, thanks for pointing this out!